### PR TITLE
Added new team members

### PR DESCRIPTION
**Pull request by mariana-kikloki**
Nova lista de membros da direção do CeSIUM 2018/19

### CODE REVIEW

**Comment by 0urobor0s** [comment]
Also check #177

File: public/team.json [hunk 1]
```diff
@@ -122,26 +132,26 @@
...
       {
-      "Marcelo Miranda": {
+      "Rui Oliveira": {
         "position": "Secretário",
-        "image": "team/marcelomiranda.jpg"
+        "image": "team/ruioliveira.jpg"
       }}
   ]
-}
+}
```
Comment (ZePedroResende) [inline public/team.json]: Add an empty line

Comment (ZePedroResende) [reply public/team.json]: You also need to compress the images

Comment (mariana-kikloki) [reply public/team.json]: I did compress the images prior to commiting them

File: public/team.json [hunk 2]
```diff
@@ -1,118 +1,128 @@
...
       {
-      "Adriana Guedes": {
-        "position": "Codiretora",
-        "image": "team/adrianaguedes.jpg"
+      "Tiago Monteiro": {
+        "position": "Codiretor",
+        "image": "team/tiagomonteiro.jpg"
       }}
-  ],
+  ]
```
Comment (0urobor0s) [inline public/team.json]: Missing comma: ```,```

File: public/team.json [hunk 3]
```diff
@@ -1,118 +1,128 @@
...
       }}
   ],
-  "DevTeams": [
+  "CAOS": [
     {
-      "Francisco Lira": {
+      "Daniel Tinoco": {
         "position": "Diretor",
-        "image": "team/franciscolira.jpg"
+        "image": "danieltinoco.jpg"
```
Comment (0urobor0s) [inline public/team.json]: You need to change this to ```team/danieltinoco``` and change the image to ```danieltinoco```

**Pull request merged**